tests/Http/Controllers/Api/Projects/FilterImageAnnotationsByLabelControllerTest.php

```diff
@@ -74,4 +79,53 @@ public function testIndexDuplicate()
             ->assertStatus(200)
             ->assertExactJson([$a1->id => $image->uuid]);
     }
+
+    public function testGetProjectsAnnotationLabels()
+    {
+        $vol = VolumeTest::create();
+        $img = ImageTest::create(['volume_id' => $vol->id]);
+        $a = ImageAnnotationTest::create(['image_id' => $img]);
+        $l = LabelTest::create();
+        $expectedLabel = [...$l->get()->toArray()[0], "uuid" => $l->uuid, "count" => 2];
+        ImageAnnotationLabelTest::create(['annotation_id' => $a->id, 'label_id' => $l->id]);
+        ImageAnnotationLabelTest::create(['annotation_id' => $a->id, 'label_id' => $l->id]);
+
+        $this->project()->volumes()->attach($vol);
+        $pid = $this->project()->id;
+
+        $this->doTestApiRoute('GET', "/api/v1/projects/{$pid}/image-annotations/");
+
+        $this->beUser();
+        $this->getJson("/api/v1/projects/{$pid}/image-annotations/")
+            ->assertStatus(403);
+
+        $this->beEditor();
+        $response = $this->getJson("/api/v1/projects/{$pid}/image-annotations/")->assertStatus(200);
+        $content = json_decode($response->getContent())[0];
```

Comment:
Have a look at the Laravel docs for [testing JSON APIs](https://laravel.com/docs/11.x/http-tests#testing-json-apis), this may be a little more elegant here.